package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "scripts": {
     "benchmark": "yarn build:cjs && yarn build:esm && jest --config benchmark.config.json",
```

Comment:
Can you remove the `build` scripts from here too? You'll need to include `benchmark` in the `react` tsconfigs, and add some mappers in

Reply:
Thanks will do that